plugin.video.youtube/resources/lib/youtube_authentication.py

```diff
@@ -7,16 +7,24 @@
     See LICENSES/GPL-2.0-only for more information.
 """
 
-from youtube_plugin.youtube.provider import Provider
-from youtube_plugin.kodion.impl import Context
+from __future__ import absolute_import, division, unicode_literals
```

Comment:
You don't need this in Python 3.

Reply:
The plugin now has a common codebase for Kodi v18+, was just easier to keep stuff like this in and change the `addon.xml` when making releases, rather than maintaining different branches for different versions of Kodi.

Can remove it for future releases, but would prefer not to, if it is not an issue.